packs/feat-effects/effect-staggering-fire.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Staggering Fire",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Staggering Fire]</p>\n<p>The target takes a -5-foot circumstance penalty to its Speed until the end of its turn, or a -10-foot circumstance penalty on a critical hit.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Staggering Fire]</p>\n<p>You take a -5-foot circumstance penalty to your Speed, or a -10-foot circumstance penalty on a critical hit.</p>"
```

Comment:
Same here. Maybe we just review all of the ones with just speed and clarify them

Reply:
Sounds like an plan